gp-ecommerce-fields/gpecf-set-discount-amount-by-field-value.php

```diff
@@ -107,6 +107,15 @@ public function output_script() {
 							value = value.split( '|' )[0];
 						}
 						self.$getInput( self.discountFieldId ).data( 'amount', value );
+						var amount = parseFloat(value, 10);
+						if ( !isNaN( amount ) ) {
+							var currency = new window.Currency(
+							window.gf_global.gf_currency_config
+						);
+
+						var formattedAmount = currency.toMoney(amount);
+							$( '.ginput_discount_' + self.formId + '_' + self.discountFieldId ).html( formattedAmount );
+						}
```

Comment:
The indention is a little wonky here. Can you make sure this is all formatted correctly?

I'm mildly surprised PHPCS isn't saying anything 😃